ezsnmp/ez.py

```diff
@@ -1,9 +1,14 @@
 from __future__ import unicode_literals, absolute_import
 
+from typing import Union, List, Tuple, Any
+
 from .session import Session
 
 
-def snmp_get(oids, **session_kargs):
+def snmp_get(
```

Comment:
Since we are going through all this trouble to specify types for the input arguments, let's also specify the return type too.

- [ ] specify the return type for all functions.